2024/survey.yaml

```
@@ -356,6 +356,19 @@ questions:
       - Wiki (unofficial)
       - Blog posts
       - Others
+  - prompt: If you want to contribute or have contributed in the past to Nixpkgs, please select all that apply:
+    type: multiple
+    choices:
+      - I don't plan to contribute.
+      - Everything I want to contribute is already being worked on and I don't feel the need to get involved
+      - I don't know how to get started.
+      - My experience contributing was as expected.
+      - I tried to contribute something but found it hard to get started.
+      - I tried to contribute, but received unhelpful feedback.
```

We have options for "tried to contribute" and options for "I was able to contribute", so let's remove "I tried to contribute" here and add a few more options. Didn't add a positive option because I didn't feel a need to, but we can add it too.

```suggestion
      - I received unhelpful feedback.
      - It took longer than I expected to receive feedback.
```